app/src/main/java/app/src/main/java/CliGameModel.java

```diff
@@ -0,0 +1,537 @@
+package app.src.main.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+import java.io.IOException; // For handling IOException if needed
+
+
+import app.src.main.java.CliGameController;
+
+
+
+public class CliGameModel extends helpers {
+    /**
+     * This is possible points a current puzzle can have.
+     */
+    private static int possiblePoints;
+    /**
+     * This is the current puzzle word.
+     */
+    private static String baseWord;
+    /**
+     * This is the shuffled puzzle word.
+     */
+    private static String shuffleWord;
+    /**
+     * This is the total points a player has.
+     */
+    private static int totalPoints;
+    /**
+     * This is the rank of a player.
+     */
+    private static String playerRank;
+    /**
+     * This is the found words list.
+     */
+    private static List<String> foundWords;
+    /**
+     * This is the required letter for the puzzle.
+     */
+    private static char reqLetter;
+    /**
+     * This is the saved data for a game.
+     */
+    private static playerData saveFile;
+    /**
+     * This is the accepted word list.
+     */
+    private static List<String> acceptedWordList;
+    /**
+     * Initialized variables
+     * @throws FileNotFoundException
+     */
+    public CliGameModel() throws FileNotFoundException
+    {
+        
+    }
+
+    /**
```

Comment:
Again, the comments are appreciated but I personally don't see the use case. The functions have descriptive names + all functions already have comment blocks with descriptions and params put out. Again willing to compromise, just reach out!